Common++/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+add_library(
+  CommonPP
```

Comment:
Why not calling it `Common++`?

Reply:
It was not supported with previous version of cmake, but now that we require a recent version i have changed them